pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/PatchMedicalProviderRoute.java

```diff
@@ -107,6 +107,7 @@ public Object handle(
                             physicianName,
                             city,
                             state,
+                            existingMedicalProviderDto.getCountryId(),
```

Comment:
This is following a different pattern from the rest of the parameters included with the payload. Is there a particular reason for the deviation?